hbook/ntup/ntup_example_test.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2019 The go-hep Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package ntup_test
+
+import (
+	"log"
+
+	"go-hep.org/x/hep/hbook"
+	"go-hep.org/x/hep/hbook/ntup/ntcsv"
+	
+)
+
+func ExampleNtuple_scanH2D() {
+	nt, err := ntcsv.Open(
+		"testdata/simple-with-header.csv",
+		ntcsv.Comma(';'),
+		ntcsv.Header(),
+		ntcsv.Columns("v1", "v2", "v3"),
+	)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	defer nt.DB().Close()
+
+	type dataType struct {
+		i int64
+		f float64
+		s string
+	}
+
+	h := hbook.NewH2D(10, 0, 10, 10, 0, 10)
+	got, err :=  nt.ScanH2D("v1,v2",h)
+	if err != nil {
+		log.Fatalf("%s: error scanning: %v", "v1, v2, v3 ", err)
+	}
+
```

Comment:
@sbinet, I am clueless what is to be done after this. Can you shed some light on it.

Please ignore any linter or style errors, once patch is complete, I will run the checks and correct accordingly.

Thanks in advance.

Reply:
according to the doc of `ntup.Ntuple.ScanH2D`:
```
$> go doc go-hep.org/x/hep/hbook/ntup.Ntuple.ScanH2D 
func (nt *Ntuple) ScanH2D(query string, h *hbook.H2D) (*hbook.H2D, error)
    ScanH2D executes a query against the ntuple and fills the histogram with the
    results of the query. If h is nil, a (100-bins, xmin, xmax+ULP) (100-bins,
    ymin, ymax+ULP) 2d-histogram is created, where xmin, xmax and ymin,ymax are
    inferred from the content of the underlying database.
```

so, I would drop the definition of `h` and just use the simpler form:
```go
h, err := nt.ScanH2D("v1, v2", nil)
```

test for the error (as you did) and then printout the mean, rms, stddev and std-err of `h`, for both X and Y (as `h` is a 2D distribution.)
see:
- https://github.com/go-hep/hep/blob/master/hbook/h2d_test.go#L78